pycsw/wsgi.py

```diff
@@ -205,6 +207,12 @@ def get_configuration_path(process_environment, request_environment,
         )
     if not os.path.isabs(configuration_path):
         configuration_path = os.path.join(pycsw_root, configuration_path)
+
+    # brute force try default.cfg in the installation directory
+    default_cfg = os.path.join(pycsw_root, 'default.cfg')
+    if os.path.isfile(default_cfg):
```

Comment:
I think this test should be checking for the existence of `configuration_path` instead of `default_cfg`.

If `configuration_path` does not exist then assign `default.cfg` to `configuration_path`